src/lib/core-components/ProgressBar.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import quiz from './../../docs/quiz';
```

Comment:
`docs/quiz` is for a demo site. In real case, this would fail for sure. You need to access `questions` from `Core` instead.